src/app/components/screenContainer/index.tsx

```diff
@@ -6,7 +6,7 @@ import styled from 'styled-components';
 const RouteContainer = styled.div((props) => ({
   display: 'flex',
   flexDirection: 'column',
-  height: 600,
+  height: '100%',
```

Comment:
https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/109fa345-8735-4b1a-9dd9-1cbe663d5469

RouteContainer here is used also by full screen /options routes

Reply:
Should be sorted 👍 Thanks for catching